### PR TITLE
Fix Forum Loading Screen Text (#4102) 

### PR DESCRIPTION
**Pull request by chrlschwb**
fixes issue #4102

### CODE REVIEW

**Review by thesan — CHANGES_REQUESTED** [review]
@chrlschwb adding the loader for the category is definitely an improvement. But similarly to https://github.com/Joystream/pioneer/pull/4318#pullrequestreview-1376297101 it doesn't deal with the consequence of the QN being unresponsive these days.

I think checking the error returned by the hook would allow to solve this issue.

[Peek 2023-04-07 17-11.webm](https://user-images.githubusercontent.com/6571453/230633586-8ff0cf5f-3848-4ec2-840a-3b3fe8514836.webm)

**Review by traumschule — APPROVED** [review]

**Comment by chrlschwb** [comment]
I think [PR 4318](https://github.com/Joystream/pioneer/pull/4318) fixes issue #4102 as well.
The only change I added is "hasError"

File: packages/ui/src/app/pages/Forum/ForumCategory.tsx [hunk 1]
```diff
@@ -59,6 +59,14 @@ export const ForumCategory = () => {
     return <Loading />
   }
 
+  if (isLoadingCategory) {
+    return <Loading />
+  }
+
+  if (hasError) {
+    return <EmptyPagePlaceholder title="Something went wrong fetching this thread" copy="" button={null} />
```
Comment (thesan) [inline packages/ui/src/app/pages/Forum/ForumCategory.tsx:67]:
```suggestion
    return <EmptyPagePlaceholder title="Something went wrong fetching this category." copy="" button={null} />
```

**Review by thesan — CHANGES_REQUESTED** [review]

**Review by thesan — APPROVED** [review]
:+1:

**Pull request merged**